packages/patrol_cli/lib/src/commands/test.dart

```diff
@@ -339,21 +343,34 @@ See https://github.com/leancodepl/patrol/issues/1316 to learn more.
     MacOSAppOptions macos, {
     required bool uninstall,
     required Device device,
+    required bool showFlutterLogs,
+    required bool showTestSteps,
   }) async {
     Future<void> Function() action;
     Future<void> Function()? finalizer;
 
     switch (device.targetPlatform) {
       case TargetPlatform.android:
-        action = () => _androidTestBackend.execute(android, device);
+        action = () => _androidTestBackend.execute(
+              android,
+              device,
+              showFlutterLogs: showFlutterLogs,
+              showTestSteps: showTestSteps,
+              flavor: flutterOpts.flavor,
```

Comment:
Is that intended?

Reply:
Yes, I need this to generate proper android report path